### PR TITLE
Updates from bedrock

### PR DESCRIPTION
**Pull request by MozmarRobot**
From file changes in https://github.com/mozilla/bedrock/commit/cf21cf26d35f6632b19d3c117e1c86572a2515c4

### CODE REVIEW

File: en/navigation_refresh.ftl [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+navigation-refresh-mozilla = { -brand-name-mozilla }
+
+## About us
+
+navigation-refresh-about-us = About us
+navigation-refresh-close-about-us-menu = Close About us menu
+navigation-refresh-mozilla-manifesto = { -brand-name-mozilla } manifesto
+navigation-refresh-mozilla-foundation = { -brand-name-mozilla-foundation }
+navigation-refresh-get-involved = Get involved
+navigation-refresh-innovation-projects = Innovation Projects
```
Comment (peiying2) [inline en/navigation_refresh.ftl:14]: Should this be `Innovation projects` (lower case P) to be consistent with other list of items?

**Review by peiying2 — COMMENTED** [review]
@craigcook not sure who is working on this PR, but in looking at https://www-dev.allizom.org/en-US/ - the title case or capitalization of title case is inconsistent depending on where you look. In most situation, it looks like this `Title case`. But some are not: `Firefox Blog` (under Firefox Browsers). Under `About us` where `Innovation Projects` is not consistent with the rest (MoFo is an exception in this case).  

The footer's title case seems to have capital initials like `Footer Title Case` vs. `Navigation title case` (or inconsistent within each menu).

File: en/navigation_refresh.ftl (hunk 1 above)
Comment (stephaniehobson) [reply en/navigation_refresh.ftl:14]: Sentance case is the official word going foward.

File: en/navigation_refresh.ftl [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+navigation-refresh-firefox-browsers = { -brand-name-firefox-browsers }
+navigation-refresh-close-firefox-browsers-menu = Close { -brand-name-firefox-browsers } menu
+navigation-refresh-firefox-for-desktop = { -brand-name-firefox } for Desktop
+navigation-refresh-firefox-for-android = { -brand-name-firefox } for { -brand-name-android }
+navigation-refresh-firefox-for-ios = { -brand-name-firefox } for { -brand-name-ios }
+navigation-refresh-firefox-focus = { -brand-name-firefox-focus }
+navigation-refresh-firefox-relay = { -brand-name-firefox-relay }
+navigation-refresh-all-products = All products
+navigation-refresh-firefox-blog = { -brand-name-firefox } Blog
```
Comment (peiying2) [inline en/navigation_refresh.ftl:27]: Should this be lower case B so it reads `Firefox blog`?

**Review by peiying2 — CHANGES_REQUESTED** [review]
Call out a few list items to use "sentence case". 

Do we change `Fireflx Browsers` to `Firefox browsers`?

File: en/navigation_refresh.ftl (hunk 1 above)
Comment (stephaniehobson) [reply en/navigation_refresh.ftl:14]: People are doing a bit of everything right now. @craigcook do you want take a pass on capitalization on Monday? I can if you're busy but I know you're better with speling.

Comment (peiying2) [reply en/navigation_refresh.ftl:14]: Let's get this updated soon as it's time sensitive. Anymore strings to come in the next couple of days?

Comment (peiying2) [reply en/navigation_refresh.ftl:14]: @alexgibson not sure if you can help fix this. We are pressing for time. If this is the last update before next week, let's finalize it. If there are more things to come, can we bundle them? Thank you!

Comment (alexgibson) [reply en/navigation_refresh.ftl:14]: @peiying2 I will take a look here thanks for the ping

Comment (alexgibson) [reply en/navigation_refresh.ftl:14]: @peiying2 PR here that should hopefully address things: https://github.com/mozilla/bedrock/pull/15164

**Review by peiying2 — APPROVED** [review]

**Pull request merged**